src/NodeWrapper.coffee

```diff
@@ -171,7 +171,11 @@ forNodeAndChildren = (node, fn) ->
 compile = (csSource, node) ->
     compiled = coffeeScript.nodes(csSource)
 
-    line = node.locationData.first_line
+    line = if node.locationData?
+      node.locationData.first_line
+    else
+      throw new Error
```

Comment:
I don't understand this fix - doesn't this `throw new Error` stop us from reaching the next line?  O_o

Reply:
correct, silly line (177).
i removed it.

Reply:
made a git mistake so i will reopen the pr.